deploy/complete/terraform/mushop-oci-services.tf

```diff
@@ -43,6 +43,28 @@ resource "kubernetes_secret" "oadb-admin" {
   count = var.mushop_mock_mode_all ? 0 : 1
 }
 
+resource "kubernetes_config_map" "oci-deployment" {
+  metadata {
+    name      = var.oci_deployment
+    namespace = kubernetes_namespace.mushop_namespace.id
+  }
+  data = {
+    region        = var.region
+    compartment_id = local.oke_compartment_ocid
+  }
+}
+
+resource "kubernetes_secret" "oapm-connection" {
+  metadata {
+    name      = var.apm_connection_name
+    namespace = kubernetes_namespace.mushop_namespace.id
+  }
+  data = {
+    zipkin_url        = "https://aaaac6gzqsj7waaaaaaaaaafse.apm-agt.us-ashburn-1.oci.oraclecloud.com"
```

Comment:
I keep this for later to create APM domain as part of terraform script, for now it's hardcoded.